src/ape/api/providers.py

```diff
@@ -291,12 +291,12 @@ def get_transaction(self, txn_hash: str) -> ReceiptAPI:
         """
 
     @abstractmethod
```

Comment:
If we make this `@raises_not_implemented()`, it will prevent it from being a breaking change on the next release

Reply:
Should eventually be `@abstractmethod` though, just add a TODO to change this after 0.2.8 release